app/crontab

```diff
@@ -0,0 +1,2 @@
+# Run every hour 
+0 * * * * python gradescope_to_spreadsheet.py >> /var/log/cron.log 2>&1
```

Comment:
Later, do some tests to see how long the script takes for CS10 Fall 2024, since there are more students in Fall 2024 compared to Summer 2024.

Reply:
As in, once an hour might be too frequent and cron job 1 might not be finished when cron job 2 starts. Make sure to have a good number in terms of how often this script reruns.